chatkit-core/src/main/kotlin/com/pusher/chatkit/messages/multipart/request/Message.kt

```diff
@@ -1,28 +0,0 @@
-package com.pusher.chatkit.messages.multipart.request
-
-import com.pusher.chatkit.CustomData
-
-data class Message(
```

Comment:
I think we should still keep this file. Probably just renaming and repackaging it to `messages.api.SendMessageRequest.kt`. Internally some renaming would also make its purpose cleaner, e.g. `SendMessageRequest`, `SendMessageRequestPart`

Reply:
👍 have put back and refactored!